src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateTFMRule.cs

```diff
@@ -75,20 +75,9 @@ private static string GetDisplayVersion(Version version)
             return sb.ToString();
         }
 
-        // TODO: When we have this inferred in the sdk targets, we won't need this
-        private AddPropertyTransform<NuGetFramework> OutputPathTransform =>
-            new AddPropertyTransform<NuGetFramework>("OutputPath",
-                f => $"bin/$(Configuration)/{f.GetShortFolderName()}",
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkIdentifierTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkIdentifier",
-                f => f.Framework,
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkVersionTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkVersion",
-                f => "v" + GetDisplayVersion(f.Version),
-                f => true);
+        private AddPropertyTransform<IEnumerable<NuGetFramework>> FrameworksTransform =>
+            new AddPropertyTransform<IEnumerable<NuGetFramework>>("TargetFrameworks",
+                frameworks => string.Join(";", frameworks.Select(f => f.GetShortFolderName())),
```

Comment:
There will be some target frameworks (PLIB profiles) for which you'll need to put the full name and set another property based on a condition of that full name to the short name. It's fine if it doesn't work at first, but we should track that with a bug.